gapis/gfxapi/vulkan/replay.go

```diff
@@ -128,6 +128,40 @@ func (t *makeAttachementReadable) Transform(ctx context.Context, id atom.ID, a a
 			out.MutateAndWrite(ctx, id, newAtom)
 			return
 		}
+	} else if recreateImage, ok := a.(*RecreateImage); ok {
```

Comment:
By all means, keep as is, as this isn't part of this CL - but it would be nice if this was cleaned up.